cosmos_transfer1/diffusion/datasets/augmentors/control_input.py

```
@@ -764,8 +772,16 @@ def __call__(self, data_dict) -> dict:
         if "control_input_lidar" in data_dict:
             # already processed
             return data_dict
-        else:
-            raise ValueError("LiDAR condition is not provided.")
+        if "video" not in data_dict:
+            image = np.array(data_dict[self.input_keys[1]])
+            h, w, _ = image.shape
+            data_dict[self.output_keys[0]] = torch.from_numpy(np.zeros((3, h, w)).astype(np.uint8))
```

Should we raise an exception since we don't currently support image input?